config/initializers/raven.rb

```diff
@@ -9,4 +9,7 @@
   config.processors << Sentry::Processor::PIISanitizer
 
   config.excluded_exceptions += ['Sentry::IgnoredError']
+  config.async = lambda { |event|
+    SentryJob.perform_async(event)
```

Comment:
can you change this to `SentryJob.perform_async(event.to_hash)`? you're not supposed to pass non-JSON serializable objects as arguments to a sidekiq job

Reply:
I looked into that earlier.  `event` is already a hash in raven 2+.

Just to sure, I added `puts event.class` on line 13 and sure enough, it is `Hash`